internal/pkg/runtime/engine/fakeroot/engine_linux.go

```diff
@@ -219,11 +219,15 @@ func (e *EngineOperations) StartProcess(masterConn net.Conn) error {
 		return fmt.Errorf("while parsing %s: %s", mountInfo, err)
 	}
 	for _, m := range mounts["/sys"] {
+		// In Linux <5.9 this is required so that in the chroot, selinux is seen as ro, i.e.
+		// disabled, and errors getting security labels do not occur.
+		// In 5.9 the remount will now fail, but it is not needed due to changes in label handling.
```

Comment:
Just curious if it's now failing because `/sys/fs/selinux` is already mounted read-only ?

Reply:
No...

```
dave@dev-fedora~> cat fakeroot.def 
Bootstrap: docker
From: alpine

%post
    mount | grep selinux

dave@dev-fedora~> singularity build --fakeroot fakeroot.sif fakeroot.def
INFO:    Starting build...
Getting image source signatures
Copying blob 188c0c94c7c5 [--------------------------------------] 0.0b / 0.0b
Copying config 9785808968 done  
Writing manifest to image destination
Storing signatures
2020/12/01 08:43:31  info unpack layer: sha256:188c0c94c7c576fff0792aca7ec73d67a2f7f4cb3a6e53a84559337260b36964
INFO:    Running post scriptlet
+ mount
+ grep selinux
selinuxfs on /sys/fs/selinux type selinuxfs (rw,nosuid,noexec,relatime)
INFO:    Creating SIF file...
INFO:    Build complete: fakeroot.sif
```

Reply:
```
DEBUG   [U=0,P=1]          StageTwo()                    Entering stage 2
DEBUG   [U=0,P=1]          StartProcess()                while remount /sys/fs/selinux read-only: operation not permitted
DEBUG   [U=0,P=1]          StartProcess()                note remount failure is expected on kernel 5.9+
```